roles/subscription-manager/tasks/offline.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: get repo file
```

Comment:
gets repo file from url instead of releying on being updated by another playbook.